server/boot/certificate.js

```diff
@@ -145,6 +196,10 @@ export default function certificate(app) {
     ifNoSuperBlock404,
     verifyCert
   );
+  router.get(
+    '/c/:username/:cert',
```

Comment:
@QuincyLarson should we use a full `certificates` here (ref: SEO)?

Reply:
I shortened it to `/c/` due to the really long `superBlock` names we have now. I wasn't sure if 
1. The URL would wrap on the cert
1. If we wanted the URL to wrap on the cert

Reply:
@Bouncey Even though it's long, I think we should spell out the word "certificates" here.